.rspec

```diff
@@ -1,3 +1,3 @@
 --color
 --require spec_helper
---format progress
\ No newline at end of file
+--format Fuubar
```

Comment:
I did this mostly for better output. I like the fact it covered unknowns as well as Pass/Fail